build.go

```diff
@@ -176,7 +176,7 @@ func (Test) Unit() error {
 	mg.Deps(Build.BuildContainer)
 	fmt.Println("\n=====> Running Satellite Unit Tests...\n")
 	return trace.Wrap(sh.RunV(
-		"docker", "run", "-it", "--rm=true",
+		"docker", "run", "-t", "--rm=true",
```

Comment:
If using the `interactive` mode, terminal gets messed up if the tests/linter fail.